librz/core/cmd_eval.c

```diff
@@ -741,7 +566,35 @@ RZ_IPI RzCmdStatus rz_eval_readonly_handler(RzCore *core, int argc, const char *
 
 RZ_IPI RzCmdStatus rz_eval_spaces_handler(RzCore *core, int argc, const char **argv) {
```

Comment:
this does not work for e.g. `es asm.sub`.

Reply:
What should `es asm.sub` return, actually?
It doesn't seem to be returning anything even on `dev`?

Reply:
It's broken also on dev indeed. `es asm.sub` should return the subkeys of `asm.sub`, the same way as `es zoom` returns the subkeys of zoom.

Anyway, let's ignore this thing for now and do it another time.